### ISSUE TITLE
setLoadingState is not a function

### ISSUE THREAD

**Issue by jeppe-smith** [matfish2/vue-tables-2#899]
- Vue.js version: 2.6.11
- consumed using: webpack
- FULL error message (including stack trace):
```
TypeError: this.$refs.clientTable.setLoadingState is not a function
    mounted DataTable.vue:41
    VueJS 18
    updateSession session.js:42
    wrappedMutationHandler vuex.esm.js:771
    commitIterator vuex.esm.js:407
    commit vuex.esm.js:406
    _withCommit vuex.esm.js:551
    commit vuex.esm.js:405
    boundCommit vuex.esm.js:350
    commit vuex.esm.js:723
    handleTokenReceived session.js:55
    wrappedActionHandler vuex.esm.js:778
    dispatch vuex.esm.js:457
    boundDispatch vuex.esm.js:347
    dispatch vuex.esm.js:706
    _callee2$ session.js:119
    Babel 10
    wrappedActionHandler vuex.esm.js:778
    dispatch vuex.esm.js:457
    boundDispatch vuex.esm.js:347
    dispatch vuex.esm.js:706
    attemptSilentSignIn session.js:147
    wrappedActionHandler vuex.esm.js:778
    dispatch vuex.esm.js:457
    boundDispatch vuex.esm.js:347
    dispatch vuex.esm.js:706
    mappedAction vuex.esm.js:991
    created LogInBarrier.vue:56
    VueJS 13
   vue.runtime.esm.js:1888
```
- steps for reproducing the issue:
Follow the documentation steps for setting loading state https://matanya.gitbook.io/vue-tables-2/client-table/asynchronous-loading

**Comment by degerstrom** [comment]
I am having the same problem. We are on the "premium features" version, and the package we are being served from GitHub does not have this function name in the code. Is there a delay between the open source and commercial releases?

**Comment by matfish2** [comment]
unable to reproduce. If you are using the premium version try to remove node_modules and reinstall to pull the latest

**Comment by degerstrom** [comment]
Sorry @matfish2, I should of course have tried that first. Resolves my issue.

**Comment by matfish2** [comment]
@jeppe-smith please check you are using the latest version

**Comment by jeppe-smith** [comment]
Resolved with version 2.1.46. Had to delete both node_modules and yarn.lock of course.
